config/config.go

```diff
@@ -101,3 +101,11 @@ func (c *Config) ProcessOptions(options ...Option) {
 		}
 	}
 }
+
+// IsValidName validates configuration objects supported by libnetwork
+func IsValidName(name string) bool {
```

Comment:
Thanks, this is better and we can have a general regex later for all the libnetwork resources' names, not only for networks.